### ISSUE TITLE
Null coalescing is not correctly handled by TC

### ISSUE THREAD

**Issue by AlexanderZhig** [DevExpress/testcafe#6995]
### What is your Scenario?

I take TC version 1.18.6, that uses testcafe-hammerhead": "24.5.18" and the problem still present:
I get the following code
d=(t&&t["function.from"])??null
converted to
d=t&&t["function.from"]??null
that leads to
Uncaught SyntaxError: Unexpected token '??'

### What is the Current behavior?

I take TC version 1.18.6, that uses testcafe-hammerhead": "24.5.18" and the problem still present:
I get the following code
d=(t&&t["function.from"])??null
converted to
d=t&&t["function.from"]??null
that leads to
Uncaught SyntaxError: Unexpected token '??'

### What is the Expected behavior?

No errors

### What is your public website URL? (or attach your complete example)

-

### What is your TestCafe test code?

d=(t&&t["function.from"])??null

### Steps to Reproduce

the following code
d=(t&&t["function.from"])??null
converted to
d=t&&t["function.from"]??null
that leads to
Uncaught SyntaxError: Unexpected token '??'

### TestCafe version

1.18.6

### Command-line arguments

-

**Comment by miherlosev** [comment]
Hi @AlexanderZhig,

I cannot reproduce the issue using this example.
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>
    <script>
        var t = {};
        t["function.from"] = 1;

        d=(t&&t["function.from"])??null;

        console.log(d)
    </script>
</body>
</html>
```

```js
fixture `Fixture`
    .page('./index.html')

test('test', async t => {
    const { log } = await t.getBrowserConsoleMessages();

    await t.expect(log[0]).eql('1');
});
```

Could you please modify my example or create your own so that we can reproduce the issue on our side?

**Comment by AlexanderZhig** [comment]
Hi Mikhail!
The following page doesn't work for me:

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title X</title>
</head>
<body>
    <script>
       const t = {};
       const {x: y} = {},
        a = (t && t["in"]) ?? null;              
    </script>
</body>
</html>
```

Just to clarify, the code above (I removed as much as possible) is a result of transpilation of the following code:
```
const Count = ({errors}) => {
  const {predicateActions} = useContext(PredicateBuilderContext);
  const i18n = useTranslations();
  const valueError = (errors && errors['in']) ?? null;
```
  ...

**Comment by miherlosev** [comment]
Thank for the shared example. I've reproduced the issue.

**Comment by theogravity** [comment]
Also encountered this as well.

**Comment by alex-zh** [comment]
Any updates on this? 
We still cannot normally run tests using TestCafe, we don't control the code that is incorrectly modified by TestCafe, so we cannot fix it on our own

**Comment by AndreyBelym** [comment]
Fixed in v1.20.0.
